kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -10,6 +10,19 @@
     model.synced = true;
     var collection = {{kolibri}}.resources.ContentNodeResource.getCollection({parent: root_node_pk}, JSON.parse("{{ nodes|escapejs }}"));
     collection.synced = true;
+
+    // Get list of channels.
+    var channels = JSON.parse("{{ channels | escapejs }}");
+    var defaultChannelId = "7199dde6-95db-4ee4-ab39-2222d5af1e51"; // TODO: API also fetches user-selected default channel.
```

Comment:
I'm a bit confused about where the default channel ID is supposed to come from - it's hard-coded in two places. Here and https://github.com/learningequality/kolibri/pull/333/files#diff-bae452d1e37f9e753fda753d8c25990aR16

Would it be possible to have just a single-source of truth?